launcher/modManager/cmodmanager.cpp

```diff
@@ -82,6 +83,13 @@ void CModManager::loadMods()
 		{
 			boost::filesystem::path name = *CResourceHandler::get()->getResourceName(resID);
 			auto mod = JsonUtils::JsonFromFile(pathToQString(name));
+			if(!name.is_absolute())
```

Comment:
Some people might decide to store mods locally as well. Lets introduce a parameter to mod json instead

Reply:
Sorry, I'm not going to continue work. There is task, @kambala-decapitator created, all further improvements can be described there if somebody decides to work on it (see previous comments)